events/wc_frostwolf_events.txt

```diff
@@ -1086,34 +1126,293 @@ narrative_event = { #Frostwolf settled
+		73 = {
+			province_event = {
+				id = WCFW.161
+				days = 7
+			}
+		}
+	}
+}
+
+# delayer event
+province_event = {
+	id = WCFW.161
+	
+	is_triggered_only = yes
+	hide_window = yes
+	
+	trigger = {
+		d_frostwolf = {
+			holder_scope = {
+				frostwolf_leader_exists = yes
+			}
+		}
+	}
+	
+	immediate = {
+		d_stormreaver = {
+			holder_scope = {
+				if = {
+					limit = { frostwolf_chain_butcher_exists = yes }
+					narrative_event = { id = WCFW.61 }
+				}
+			}
+		}
+	}
+}
+
+# Forcer informed of failure event (this can fail if Stormreaver is dead)
+# delayed event
+narrative_event = {
+	id = WCFW.61
+	title = EVTTITLE_WCFW_61
+	desc =  EVTDESC_WCFW_61 # Despite the fairly clear orders you gave to [From.GetFirstName], the Frostwolf clan refused to leave the Horde.
+	picture = GFX_evt_orc_city
+	border = GFX_event_narrative_frame_war
+
+	is_triggered_only = yes
+
+	option = {
+		name = EVTOPT_WCFW_61 # What is necessary shall be done.
+		73 = {
+			province_event = {
+				id = WCFW.162
+				days = 7
+			}
+		}
+	}
+}
+
+# delayer event
+province_event = {
+	id = WCFW.162
 	
 	is_triggered_only = yes
+	hide_window = yes
+	
+	trigger = {
+		d_frostwolf = {
+			holder_scope = {
+				frostwolf_leader_exists = yes
+			}
+		}
+		d_stormreaver = {
+			holder_scope = {
+				frostwolf_chain_butcher_exists = yes
+			}
+		}
+	}
 	
 	immediate = {
 		d_frostwolf = {
-			clr_title_flag = fw_hunger_event_flag
+			holder_scope = {
+				save_event_target_as = target_frostwolf_leader
+			}
+		}
+		
+		any_playable_ruler = {
+			limit = {
+				OR = { # ((Part of the Horde AND player) OR Frostwolf Leader)
+					AND = {
+						ai = no
+						realm = { title = e_horde }
+					}
+					character = event_target:target_frostwolf_leader
+				}
+			}
+			narrative_event = { id = WCFW.62 }
 		}
 	}
+}
 
+# BUTCHERING EVENT
+# Delayed event
+narrative_event = {
+	id = WCFW.62
+	title = EVTTITLE_WCFW_62
+	
+	immediate = {
+		d_frostwolf = {
+			holder_scope = {
+				save_event_target_as = target_frostwolf_leader
+			}
+		}
+		d_stormreaver = {
+			holder_scope = {
+				save_event_target_as = target_frostwolf_butcher
+				
+				# Create the new character under Stormreaver
+				create_character = {
+					dynasty = actually_culture
+					culture = stormreaver religion = orcish_fel
+					trait=tough_soldier
+					trait=creature_orc
+					trait=class_warrior_5
+					random_traits = yes
+					age = 22
+				}
+				new_character = {
+					save_event_target_as = target_new_orc
+				}
+			}
+		}
+	}
+	
+	desc = {
+		trigger = { character = event_target:target_frostwolf_leader }
+		text = EVTDESC_WCFW_62 # You are outside watching the campfire when you suddenly hear loud screams...
+	}
+	desc = {
+		trigger = { NOT = { character = event_target:target_frostwolf_leader } }
+		text = EVTDESC_WCFW_62_horde # Word arrives that the Frostwolf clan led by [target_frostwolf_leader.GetFirstName] has been completely decimated...
+	}
```

Comment:
> Not really a problem, but maybe another description in the case if butcher himself receive this event would be look better? Something like what you did in #768

Same here.

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46576860/69394619-8799af80-0ced-11ea-9940-9bd08aafec03.png)

</details>